apps/pdc-frontend/src/components/MatomoTagManager.tsx

```diff
@@ -0,0 +1,32 @@
+'use clint';
+
+import Script from 'next/script';
+import type { ScriptProps } from 'next/script';
+
+type MatomoTagManagerTypes = {
+  [key: string]: any;
+};
+declare global {
+  // eslint-disable-next-line no-unused-vars
+  interface Window {
+    _mtm: MatomoTagManagerTypes;
+  }
+}
+
+interface MatomoTagManagerProps extends ScriptProps {}
+
+export const MatomoTagManager = ({ nonce, ...restProps }: MatomoTagManagerProps) => {
+  return (
+    <Script
+      id="_next-matomo"
+      src="https://stats.utrecht.nl/analytics/js/container_Ku3GHbr3.js"
```

Comment:
Deze moeten we eigenlijk in een env var zetten bedenk ik me. De container verschilt namelijk per omgeving volgens Joram

Reply:
Bedoel je dat we de volledige link, inclusief `container_Ku3GHbr3.js`, als een env variable moeten maken?
Omdat ik denk dat deze `container_Ku3GHbr3.js` iets anders zou kunnen zijn.

Reply:
Eigenlijk alleen de container code inderdaad.